docs/consume-packages/includes/restore-nuget-exe-cli.md

```diff
@@ -1,13 +1,21 @@
-Use the [restore](../../reference/cli-reference/cli-ref-restore.md) command, which downloads and installs any packages missing from the *packages* folder.
+The NuGet CLI [restore](../../reference/cli-reference/cli-ref-restore.md) command downloads and installs any missing packages. The command works on projects that use either [PackageReference](/nuget/consume-packages/package-references-in-project-files) or [packages.config](/nuget/reference/packages-config) for package references.
 
-For projects migrated to PackageReference, use [msbuild -t:restore](../package-restore.md#restore-using-msbuild) to restore packages instead.
+Like `install`, the `restore` command only adds packages to disk, but doesn't modify the project file or *packages.config*. To add project dependencies, use the Visual Studio Package Manager UI or Console.
 
-`restore` only adds packages to disk but does not change a project's dependencies. To restore project dependencies, modify `packages.config`, then use the `restore` command.
+To restore packages, run the following command:
 
-As with the other `nuget.exe` CLI commands, first open a command line and switch to the directory that contains your project file.
+```cli
+nuget restore <projectPath>
+```
```

Comment:
I think there's value in duplicating the following here as well: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#restoring-packagereference-and-packagesconfig-projects-with-msbuild

MSBuild 16.5+ supports restoring packages.config that are in the same directory as a solution project `msbuild -t:restore -p:RestorePackagesConfig=true`.